package.json

```diff
@@ -26,9 +26,10 @@
   },
   "homepage": "https://github.com/treshugart/gulp-auto-task",
   "devDependencies": {
-    "gulp": "^3.8.11"
```

Comment:
It looks for gulp up the tree, or you can pass it in so this isn't really necessary anymore.